airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/requester.py

```diff
@@ -138,6 +138,21 @@ def request_kwargs(
         this method. Note that these options do not conflict with request-level options such as headers, request params, etc..
         """
 
+    @abstractmethod
+    def send_request(
+        self,
+        path: Optional[str] = None,
```

Comment:
when would the path be None?

Reply:
In most cases - it's only not none if the caller wants to override the "regular" path which happens for path injection in the paginator